docs/en/ingest-management/fleet/upgrade-elastic-agent.asciidoc

```diff
@@ -14,6 +14,11 @@ maintenance window during which the upgrades will occur. If your {stack}
 subscription level supports it, you can schedule upgrades to occur at a specific
 date and time.
 
+In most failure cases the {agent} may retry an upgrade after a short wait. The
+wait durations betweem retries are: 1m, 5m,  10m, 15m, 30m, 1h. During this
+time, the {agent} may show up as "retrying" in the {fleet} UI. Note that an
+upgrade that is being retried may be aborted as detailed below.
```

Comment:
Avoid positional words like "below" (a link is better because sections have a way of moving around or getting deleted). Also best to avoid passive so it's clear that the user needs to take an action here.
```suggestion
time, the {agent} may show up as "retrying" in the {fleet} UI. Note that you
can abort an upgrade that is being retried. See <<abort-agent-upgrade>>.
```